### PR TITLE
Version 0.14.0

### PR DESCRIPTION
**Pull request by FrederikBolding**
Cuts version `0.13.1`, which includes the EIP712 `v` fix.

### CODE REVIEW

File: package.json
Comment (mcmire) [inline package.json:3]: Should this be 0.13.1 instead, since it only includes a fix?

Comment (FrederikBolding) [reply package.json]: Done!

**Review by mcmire — DISMISSED** [review]
LGTM!

**Review by Gudahtt — DISMISSED** [review]
LGTM. We can discuss how to manage this change to the MetaMask provider API elsewhere; either way this release is fine.

**Comment by brad-decker** [comment]
@danjm @Gudahtt @FrederikBolding updated to include the bump of ethereumjs/tx but did not update the changelog as its not a user facing change. Is that correct?

**Comment by FrederikBolding** [comment]
> updated to include the bump of ethereumjs/tx but did not update the changelog as its not a user facing change. Is that correct?

That's what I would do.

**Comment by brad-decker** [comment]
That's a very fair point @Gudahtt -- What would you prefer.. remove that commit from this release, merge this and then prepare a 0.14.0 OR bump this 0.14.0 ?

**Comment by Gudahtt** [comment]
Good question. Maybe better to release these separately to allow for projects like mobile to get that bug fix without being blocked on ES2020/BigInt compatibility

**Comment by brad-decker** [comment]
@Gudahtt that commit was dropped from this branch, if we do the publish from it then it won't be included in the 0.13.1 release. I'll work on a 0.14.0 branch now

**Comment by Gudahtt** [comment]
Hmm, I think we'll need to revert it on main. Publishing in libraries gets triggered after merging the RC, so the revert needs to land there, not just be omitted from this branch.

**Comment by brad-decker** [comment]
publishing happens in this library automatically? I don't see github actions setup.

**Comment by brad-decker** [comment]
Reverted the commit for ethereumjs/tx update and rebased this. @Gudahtt

**Comment by Gudahtt** [comment]
It looks like we'll want this to be a breaking change release as well, because https://github.com/MetaMask/eth-ledger-bridge-keyring/pull/169 will be included

This seems OK because a minimum Node.js version of v14 doesn't present any complications for our teams though. So the split we've made still makes sense, just we'll need to shift the numbers

**Comment by mcmire** [comment]
Oops just noticed this, sorry for the bad advice earlier!

**Comment by brad-decker** [comment]
On it. 😅

**Comment by brad-decker** [comment]
@Gudahtt care to check my addition/changes to make sure this accurately reflects the changes

File: package.json
Comment (brad-decker) [reply package.json:3]: changed back to 0.14.0

File: CHANGELOG.md
Comment (Gudahtt) [inline CHANGELOG.md:11]: Nit: To match formatting we use on other repos, and to add a newline between sections as is required by our auto-changelog linting elsewhere (not sure why the linter isn't setup here :thinking: )

```suggestion
- **BREAKING:** The minimum version of Node.js required for this package has been bumped to v14.

```

File: CHANGELOG.md
Comment (Gudahtt) [inline CHANGELOG.md:11]: Nit: Add PR link

```suggestion
- **BREAKING:** The minimum version of Node.js required for this package has been bumped to v14. ([#169](https://github.com/MetaMask/eth-ledger-bridge-keyring/pull/169))
```

**Review by Gudahtt — DISMISSED** [review]
LGTM but one suggestion left in a comment

**Comment by brad-decker** [comment]
> A new version of the package @metamask/eth-ledger-bridge-keyring (0.14.0) was published at 2023-03-20T17:50:04.309Z from
[[REDACTED]]. The shasum of this package was d02f21e236d31718372706ce535fb6c56f2214ca.

**Pull request merged**
